src/Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv/Internal/ListenerContext.cs

```diff
@@ -78,8 +78,7 @@ private UvStreamHandle AcceptHandle()
             switch (EndPointInformation.HandleType)
             {
                 case FileHandleType.Auto:
-                    //  Handle type detection must be done by listener, refreshing endpoint information
-                    throw new InvalidOperationException();
+                    throw new InvalidOperationException("Cannot accept on a non-specific file handle, listen should be performed first.");
```

Comment:
Since we don't expect this to throw in practice, I wouldn't bother putting the message in a the resx file.